### PR TITLE
Feature/new protopolis screen

### PR DESCRIPTION
**Pull request by ysbrandB**
A new screen for in the Protopolis on the left TV screen. This combines the Protopolis info and the narrowcasting, as we no longer have separate screens for them.
Adds the screen and implements the necessary code for it.
It also fixes the narrowcasting.

The screen looks like this (A too long activity title now truncates not scrolls but that is not yet reflected in the screenshot):
protopolisScreen

### CODE REVIEW

File: resources/views/smartxp/protopolis_screen.blade.php [hunk 1]
```diff
@@ -0,0 +1,328 @@
...
+    </style>
+
+    </style>
+@endpush
+
+@section('body')
+    <div class="row text-black">
+
+        {{-- narrowcasting--}}
+        <div class="col-9" style="border-radius: 8px; padding:0; overflow:hidden;  border: 0px solid #00AAC0;
```
Comment (Jessseee) [inline resources/views/smartxp/protopolis_screen.blade.php:65]: Why all this inline css?

File: resources/views/smartxp/protopolis_screen.blade.php [hunk 2]
```diff
@@ -0,0 +1,328 @@
...
+                              newDiv.style.backgroundRepeat = 'no-repeat'
+                          }
+
+
+                          let titleDiv=document.createElement("div")
+                          titleDiv.className="overflow-hidden"
+
+                          let titleSpan=document.createElement("span")
+                          titleSpan.innerHTML=activity.title
+                          titleSpan.className="text-nowrap text-truncate w-100 me-5 font-weight-bold font-size-lg"
```
Comment (Jessseee) [inline resources/views/smartxp/protopolis_screen.blade.php:229]: I think a scrolling title would be preferable. See the ProTube screen for an example.

**Review by Jessseee — CHANGES_REQUESTED** [review]
LAGTM, nice design btw.

File: resources/views/smartxp/protopolis_screen.blade.php (hunk 2 above)
Comment (ysbrandB) [reply resources/views/smartxp/protopolis_screen.blade.php]: Implemented :)

**Review by Jessseee — APPROVED** [review]
LGTM

**Pull request merged**